mino/proxy/http/controller/action.go

```diff
@@ -0,0 +1,28 @@
+package controller
+
+import (
+	"fmt"
+	"time"
+
+	"go.dedis.ch/dela/cli/node"
+	"go.dedis.ch/dela/mino/proxy/http"
+)
+
+type startAction struct{}
+
+// Execute implements node.ActionTemplate. It starts and injects the proxy http
+// server.
+func (a startAction) Execute(ctx node.Context) error {
+	addr := ctx.Flags.String("clientaddr")
+
+	proxyhttp := http.NewHTTP(addr)
+
+	ctx.Injector.Inject(proxyhttp)
+
+	go proxyhttp.Listen()
+	time.Sleep(time.Second)
+
+	fmt.Fprintf(ctx.Out, "started proxy server on %s", proxyhttp.GetAddr().String())
```

Comment:
Listen can fail, here we assume that it's always going to be successful.
Shouldn't we ask of Listen() to return an error code?

Reply:
Let's add that later on a new PR if we feel it's needed. I don't want to rush into updating the proxy interface without taking the time to properly update the tests.

Reply:
Works for me.
You might want to add a comment, so that whoever revisits this code knows of that implicit assumption though.
It'll save us time when (eventually) the problem shows up :)